proto/p2p/p2p.proto

```diff
@@ -91,17 +91,17 @@ message Pong {
   repeated SubnetUptime subnet_uptimes = 2;
 }
 
-// Version is the first outbound message sent to a peer when a connection is
+// Handshake is the first outbound message sent to a peer when a connection is
 // established to start the p2p handshake.
 //
-// Peers must respond to a Version message with a PeerList message to allow the
+// Peers must respond to a Handshake message with a PeerList message to allow the
 // peer to connect to other peers in the network.
 //
 // Peers should drop connections to peers with incompatible versions.
-message Version {
+message Handshake {
```

Comment:
Probably should be a separate PR - but we should probably rename `my_version_time` in here as well.